abb_irb120_support/package.xml

```diff
@@ -0,0 +1,41 @@
+<package>
+  <name>abb_irb120_support</name>
+  <version>0.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 120
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 120 manipulator. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the ABB data sheets.  All URDFs / XACROs are based on the
+      default motion and joint velocity limits, unless noted otherwise (ie:
+      no support for high speed joints, extended / limited motion ranges or
+      other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</author>
+  <maintainer email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</maintainer>
+  <license>Apache2</license>
+
+  <url type="website">http://moveit.ros.org/</url>
```

Comment:
This is a support package, not a MoveIt package, right? `website` should probably be something like `http://wiki.ros.org/abb_irb120_support`.